packages/core/src/checkbox/checkbox.spec.ts

```diff
@@ -0,0 +1,137 @@
+import { expect, fixture } from '@open-wc/testing';
+import { sendKeys } from '@web/test-runner-commands';
+import { html } from 'lit';
+import { Checkbox } from './checkbox.js';
+import './register.js';
+
+describe('sl-checkbox', () => {
+  let el: Checkbox;
+
+  describe('defaults', () => {
+    beforeEach(async () => {
+      el = await fixture(html`<sl-checkbox>Hello world</sl-checkbox>`);
+
+      el.disabled = false;
```

Comment:
Why?

Reply:
that is quite an existential question to answer in a code review...